### PR TITLE
listOfType - for stream curation

### PR DESCRIPTION
**Pull request by andygnewman**
Some streams will have curated lists of content.
These will come from CAPI lists via a type and stream (concept) id rather than just a specific list id.

### CODE REVIEW

File: server/lib/schema.js [hunk 1]
```diff
@@ -324,6 +324,25 @@ const queryType = new GraphQLObjectType({
...
+				}
+			},
+			resolve: (root, { listType, concept }, { rootValue: { flags, backend = backendReal }}) => {
+				const be = backend(flags);
+				return be.capi.listOfType(listType, concept)
+					.then(list => {
+						const articlesUuids = list.items.map(item => item.id.substring(item.id.length - 36));
+						return be.capi.content(articlesUuids)
+					});
+			}
```
Comment (ironsidevsquincy) [inline server/lib/schema.js]: this does seem to go against the other queries in this schema, which are very specific things (opinion stories, top stories, etc). This is more a general api. Which might be fine, just feels like it's _maybe_ going against the idea of graphql. But the three queries above it are equally general, so not a blocker, just something to bear in mind

Comment (andygnewman) [inline server/lib/schema.js]: Agree and would rather have just gone straight to CAPI, but wanted the article modelling that is in GraphQL to then be able to use n-section which is inextricably linked to that model. Medium term we should abstract the article modelling out of GraphQL into it's own component so we can use independently.

Comment (ironsidevsquincy) [inline server/lib/schema.js]: Yep, make sense

**Comment by ironsidevsquincy** [comment]
👍

**Pull request merged**